### PR TITLE
Add a "RESTful" API

### PR DESCRIPTION
**Pull request by Haroenv**
* /api/1/polls -> array of all polls
* /api/1/polls/{id} -> certain poll

Both contain all of the info needed

### CODE REVIEW

File: src/main/java/com/pollistics/models/Poll.java [hunk 1]
```diff
@@ -29,7 +30,7 @@ public Poll(String name, HashMap<String,Integer> options, User user) {
 
 	public Poll() {
 	}
-
+	@JsonIgnore
```
Comment (Haroenv) [inline src/main/java/com/pollistics/models/Poll.java]: Why is this @Maliek? IMO you should be able to find the id of a poll. Maybe even better would be: 

```json
{
  "someId": {},
  "otherId": {},
  "blablabla": {}
}
```

Which is easier to use

File: src/main/java/com/pollistics/controllers/APIController.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+@RestController
+@RequestMapping("/api/1/polls") // todo: this should have a subclass or something for all poll-things
+public class APIController {
+	@Autowired
+	private PollService pollService;
+
+	public @RequestMapping(method = RequestMethod.GET)
+	List<Poll> findAll() {
+		return pollService.getAllPolls();
```
Comment (elsmr) [inline src/main/java/com/pollistics/controllers/APIController.java:22]:
```java
@GetMapping
public List<Poll> findAll()
```

looks cleaner, please change this for all methods :100:.

Comment (Haroenv) [reply src/main/java/com/pollistics/controllers/APIController.java:22]: Tip top, I'll do it for you @Maliek

**Review by Haroenv — COMMENTED** [review]
Finished.

**Review by elsmr — APPROVED** [review]
LIT :fire:

**Pull request merged**